models/ad_reporting.sql

```diff
@@ -2,7 +2,8 @@
 
 with unioned as (
 
-    {{ dbt_utils.union_relations(get_staging_files()) }}
+    {{ dbt_utils.union_relations(get_staging_files(),
+                                column_override = {"account_id": "string"} if target.type in ['bigquery', 'spark'] else {"account_id": "varchar"} ) }}
```

Comment:
ok so i kept this but changed it to strings....in certain platforms (ex: twitter) account_id's are strings (they have numbers + letters) while account_id's in FB are just numbers. adding this override fixed the `integer out of range` issue i was seeing, presumably because some stringy IDs were being forced into integers

Reply:
We chatted about this so comfortable resolving the topic. Thanks!